src/libultra_code/sptask.c

```diff
@@ -0,0 +1,66 @@
+#include "libultra_internal.h"
+#include <ultra64/hardware.h>
+
+#define _osVirtualToPhysical(ptr)              \
+    if (ptr != NULL) {                         \
+        ptr = (void*)osVirtualToPhysical(ptr); \
+    }
+
+static OSTask tmp_task;
```

Comment:
This name doesn't fit our naming scheme so it needs some adjustments.
```suggestion
static OSTask sTmpTask;

```